.gitignore

```diff
@@ -29,6 +29,7 @@ doc/_build
 doc/venv
 doc/tutorial/static/api.js
 doc/tutorial/static/jq.js
+tags
```

Comment:
this really should be part of your global gitignore. I'm against putting all editor temporary files into projects `.gitignore`.

Other stuff looks ok.